packages/core/src/utils/snapshot-progress-renderer.ts

```diff
@@ -0,0 +1,90 @@
+import { Container, Contracts } from "@arkecosystem/core-kernel";
+import { SnapshotApplicationEvents } from "@arkecosystem/core-snapshots";
+import { Ora } from "ora";
+
+export class ProgressRenderer {
+    private isAnyStarted: boolean = false;
+
+    private spinner: Ora;
+
+    private blocksProgress: string = "---.--";
+    private transactionsProgress: string = "---.--";
+    private roundsProgress: string = "---.--";
+
+    // @ts-ignore
+    private blocksCount: number = 0;
+    // @ts-ignore
+    private transactionsCount: number = 0;
+    // @ts-ignore
```

Comment:
What's the issue here that requires a `@ts-ignore`?

Reply:
I am getting and setting the counter like this: [table_name]Count = x;
Because of the indirect usage, compiler is going to say there are unused variables.

Reply:
Wouldn't it make more sense to use an object that holds the counts by their key to avoid this issue instead of silencing the compiler?

```ts
private readonly counts = {
    blocks: 0,
    transactions: 0,
    rounds: 0,
};


this.counts[tableName]++;
```

Reply:
Yes. This is way more elegant.